.github/workflows/build-hotfix.yml

```diff
@@ -65,3 +66,20 @@ jobs:
           git commit -m "chore(dependencies): Autobump oss clouddriver version"
           git pull
           git push
+      # Adding OSS docker build for SNAP
```

Comment:
This branch should be kept customer agnostic. Also this repo is public, so we can omit customer names.

Reply:
Removed name.